capstone/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,61 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import java.io.BufferedReader;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.sps.data.PerspectiveInput;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
We should modify this comment to reflect the edited server

Reply:
Fixed